changelog.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 0.9.0
+* Add transcript export and print (#1624)
+* Add batch export (#1627)
+* Increase timeout length on file downloads (#1635)
+* Localize dates shown in UI (#1636)
+* Always show "last refresh" status message (#1613) 
+* Correct license terms (#1571)
+
```

Comment:
```suggestion
## 0.9.0
* Add support for `pt_PT` and `zh_Hans` (#1531)
* Add transcript export and print (#1624)
* Add batch export (#1627)
* Adjusted expectations of Tor network speed to prevent undue download failures (#1635)
* Localize dates shown in UI (#1636)
* Improve messaging while messages are being retrieved (#1604, #1613)
* Unexpected behavior of sent replies after network connection failures (#1486)
* Fix desktop entry to follow Freedesktop specification (#1589)
* Remove tainted strings from UI and production logs (#1559, #1560)
* (Dev) Improve support for development on MacOS X (#1590, #1592, #1593, #1603)
* (Dev) Fix numerous developer-facing typos (#1588)
```

Added a few important fixes (e.g. #1486), made sure all the external contributions are listed (#1636, #1604, #1613, #1589, #1592, #1593, #1588).
I also rephrased the timeout entry from a concrete (what does it mean for me?) journalist point of view.
And removed the license documentation fix that is pretty minor in comparison to other stuff we're leaving out.

Reply:
awesome ty, keeping all your changes except the license one since it's good to document when those changes happen. Will update presently

Reply:
Darn:

```diff
- * Unexpected behavior of sent replies after network connection failures (#1428 
+ * Fix unexpected behavior of sent replies after network connection failures (#1486)
```

Reply:
i gotcha don't worry :)